lms/djangoapps/certificates/signals.py

```diff
@@ -68,16 +68,22 @@ def _listen_for_track_change(sender, user, **kwargs):  # pylint: disable=unused-
 
     user_enrollments = CourseEnrollment.enrollments_for_user(user=user)
     grade_factory = CourseGradeFactory()
+    expected_verification_status, _ = SoftwareSecurePhotoVerification.user_status(user)
```

Comment:
Can you update the name of this method?  This is not fired when the learner's track changes.  Instead, this is fired when the learner's id-verification-status changes.